functions/APIs/deals.js

```diff
@@ -1,11 +1,14 @@
+const express = require("express");
+const dealsRoute = express.Router();
+const auth = require("../util/auth");
 const { db } = require("../util/admin");
 
 /*
 read a list of all deals
 GET /deals
 success response: array of deal objects (documented in POST /deals)
 */
-exports.getAllDeals = (request, response) => {
+dealsRoute.get("/", auth, (req, res) => {
```

Comment:
nit: maybe `request` and `response` would be more readable. that's what they were before? i prefer the old way

Reply:
I changed request and response to req and res because the Firebase [documentation](https://firebase.google.com/docs/functions/http-events) uses req and res.

Reply:
I still prefer the full names. The improvement in readability is worth the very slight increase in binary size (and most production apps minify code before a user can get to it anyway)
But it's not a deal breaker.